src/CloudFoundry.VisualStudio/PublishXmlEditorFactory.cs

```diff
@@ -15,6 +15,7 @@ public int Close()
             return VSConstants.S_OK;
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes", Justification = "Prevent VS crash"), System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Globalization", "CA1308:NormalizeStringsToUppercase", Justification = "Extension is lower case")]
```

Comment:
For the string casing - it's complaining because comparison with low may not be accurate (see [here](https://msdn.microsoft.com/en-us/library/bb386042.aspx)).
You can change the if statement to work with uppercase - that doesn't mean the extension needs to be uppercase.